CHANGELOG.md

```diff
@@ -18,10 +18,9 @@ Valid subsections within a version are:
 
 Things to be included in the next release go here.
 
-### Added
+### Fixed
 
-- `collectgarbage()` is now called during cleanup of `TSPDevice` children.
```

Comment:
Why did you remove these entries from the changelog?

Reply:
Not intentional. Hopefully those entries are back and everything is good.